meta/main.yml

```diff
@@ -0,0 +1,32 @@
+galaxy_info:
+  author: Janne Heß
+  description: Install and configure a clamav daemon with freshclam
+  company: stuvus (http://stuvus.uni-stuttgart.de)
```

Comment:
Are you not using our latest ansible skeleton?

Reply:
I obviously don't